exercises/practice/raindrops/.approaches/introduction.md

```diff
@@ -0,0 +1,91 @@
+# Introduction
+
+Raindrops exercise is a twist of the popular `FizzBuzz` problem.
```

Comment:
```suggestion
Raindrops exercise is a twist on the popular `FizzBuzz` problem.
```

Also perhaps there should be a link to what FizzBuzz is if the reader doesn't know the term.